lib/Service/ImportService.php

```diff
@@ -16,7 +16,7 @@
 use \OCA\News\Db\Item;
 use \OCA\News\Db\Feed;
 
-use \Psr\Log\LoggerInterface;
+use Psr\Log\LoggerInterface;
```

Comment:
This change seems unnecessary

Reply:
Yea true, probably changed it because of some error due to another reason..